### ISSUE TITLE
Call for GUI coders: create an intuitive GUI to drive RP2 and DaLI

### ISSUE THREAD

**Issue by eprbell** [eprbell/rp2#93]
This would be a separate Github project. It would be GUI-based and provide intuitive operation for people who aren't power users or comfortable using a CLI interface. The GUI would collect input from the user, and drive DaLI and RP2 to generate the user's taxes. The main goal would be to build an easy-to-use interface for non-technical people, similar to commercial crypto tax software.

**Comment by alibama** [comment]
yeah = i'm in = let me help you put this in to streamlit and we'll have a baller product in a couple of weeks

anson@virginia.edu ping me?

**Comment by alibama** [comment]
and maybe https://trymito.io for an intermediary product?

**Comment by eprbell** [comment]
Thanks for volunteering to help with an external project! The user interface for RP2 and DaLI is fairly well defined, so hopefully you can drive them easily from your GUI. Looking forward to see what you come up with. If you have questions let me know.

**Comment by alfbrown** [comment]
This would be an amazing feature to add to my existing site! Would welcome the opportunity to discuss this is a possibility. I reached out to you on X.
